kafka/datadog_checks/kafka/data/metrics.yaml

```
@@ -411,7 +411,7 @@ jmx_metrics:
             alias: kafka.request.produce.failed.rate
     - include:
         domain: 'kafka.network'
-        bean: 'kafka.network:type=RequestMetrics,name=RequestsPerSec,request=Produce'
+        bean_regex: 'kafka.network:type=RequestMetrics,name=RequestsPerSec,request=Produce.*'
```

To reviewer: Is it better just to create a new metric for this and specify `version=.*` instead? This may be an overcorrection.

---

I think this is fine since the bean_regex will match the original metric bean name and also the newer ones with additional tags. 

What is your reasoning for making it a new metric?

---

I wasn't sure if directly appending `.*` at the end is too much of a change. If for some reason there's yet another change to `RequestsPerSec` metrics that adds other information but would be more suitable as a separate metric, then we wouldn't have any way to resolve that. However, I'm not sure how common that is or if we had that happen in the past.

---

That makes sense! How about we make the regex more specific? 

```suggestion
        bean_regex: 'kafka.network:type=RequestMetrics,name=RequestsPerSec,request=Produce(?:,version=.*)?'
```
This pattern should match the original bean name and the new bean name as long as the only diff is the added version label